app/javascript/initializeEasyMDE.js

```diff
@@ -0,0 +1,19 @@
+const events = ['turbo:load', 'turbo:render']
+
+for (const event of events) {
+  document.addEventListener(event, function () {
+    const textArea = document.querySelector(".annotation_guideline_textarea")
+
+    if (textArea && !textArea.dataset.easyMDEInitialized) {
+      // Set flag to avoid double initialization.
+      textArea.dataset.easyMDEInitialized = "true"
+
+      new EasyMDE({
+        element: textArea,
+        spellChecker: false,
+        toolbar: ["bold", "italic", "heading", "|", "quote", "code", "unordered-list", "ordered-list", "|", "preview", "side-by-side", "fullscreen"]
```

Comment:
```suggestion
        toolbar: [
          "bold",
          "italic",
          "heading", 
          "|", 
          "quote", 
          "code", 
          "unordered-list", 
          "ordered-list", 
          "|", 
          "preview", 
          "side-by-side", 
          "fullscreen"
        ]
```
パラメータがたくさんあるので、縦に並べて欲しいです。

Reply:
修正しました。
https://github.com/yush-nh/annotation_guidelines/pull/26/commits/ae88ff479e7f101115903e99c37324a43e531af2